#used so that a 2D array of floats can be sent in a message
float64[] values
